app/workers/evss/disability_compensation_form/submit_form0781.rb

```diff
@@ -167,8 +177,9 @@ def perform_client_upload(file_body, document_data)
       end
 
       def client
+        # [wipn8923] is this ever called when running the LH process??
         @client ||= if Flipper.enabled?(:disability_compensation_lighthouse_document_service_provider)
-                      # TODO: create client from lighthouse document service
+                      BenefitsDocuments::Service.new(submission.auth_headers)
```

Comment:
We found out this was incorrect after investigating this service and discovering it was tailored to the mobile team's use case only.

Changes in this file will be very similar to the changes I made to the Submit Upload worker: https://github.com/department-of-veterans-affairs/vets-api/pull/13835/commits/2fb7784c80147a0aaa84d647c4f6fbcbeadfa1ee

Actually seeing some shared behavior between these we can toss into a module somewhere